src/style.css

```diff
@@ -0,0 +1,305 @@
+/*************************/
+/* HEADER*/
+/*************************/
```

Comment:
Consider creating different css files for each of these, as opposed to putting all the styling into one file. For now it's not too long so it's not too difficult to look through, especially with the comments you've added, but in the future this may become very crowded as you add more components.